pkgs/development/python-modules/mawk/default.nix

```
@@ -0,0 +1,28 @@
+{
+  python3Packages,
+  buildPythonPackage,
+  lib,
+}:
+
+buildPythonPackage rec {
+  pname = "mawk";
+  version = "0.1.4";
+  src = python3Packages.fetchPypi {
```

```suggestion
  src = fetchPypi {
```
In python-modules it is not allowed to use the final package set.